scipy/stats/stats.py

```diff
@@ -7141,8 +7188,7 @@ def kruskal(*args, **kwargs):
        The Kruskal-Wallis H statistic, corrected for ties.
     pvalue : float
        The p-value for the test using the assumption that H has a chi
-       square distribution. The p-value returned is the survival function of
-       the chi square distribution evaluated at H.
+       square distribution.
```

Comment:
Looks like I might have forgotten to pull before commiting...? Either way, unintentional. Is there any easy way to revert the change or should I just re-commit the doc?

Reply:
Yeah I think just add it back. You can either do that manually on your own computer, but I want to suggest another way just so you get familiar with using GitHub.

You should see a button that will let you accept this suggestion:
```suggestion
       square distribution. The p-value returned is the survival function of
       the chi square distribution evaluated at H.
```
Then on your machine:
```
git fetch origin
git status
```
Will get the latest information from your remote (`origin`) and tell you the `status` of your local branch relative to the one on the remote. It will tell you that the remote branch is one commit ahead. It might also tell you you have local changes, in which case you should commit (or stash) them before you rebase on your remote branch:
```
git rebase origin/ttest-single-tailed-pvalues
```
(Which is similar to "merging" the changes, but [works a bit differently](https://www.atlassian.com/git/tutorials/merging-vs-rebasing).) 
If you were to accept these changes online without rebasing (or merging), the next time you try to push to the remote it would get an error like [this](https://happygitwithr.com/push-rejected.html).